### ISSUE TITLE
Timber.JS - 'Prevent touchstart' on Nav causing accessibility issues 

### ISSUE THREAD

**Issue by admanami** [Shopify/Timber#520]
The following function in `/assets/timber.js.liquid ` is causing keyboard navigation (tabbing) to fail on the `#AccessibleNav` element whenever it encounters a drop-down menu (`.site-nav--has-dropdown`).

```
 $subMenuLinks.on('touchstart', function(evt) {
    // Prevent touchstart on body from firing instead of link
    evt.stopImmediatePropagation();
  });
```

**Expected behaviour:**
Using the keyboard, you should be able to tab from element to element all the way down the page (or in tabindex order).

**Actual behaviour:**
Tabbing works _**until**_ a drop down menu is encountered, then the tabbing is refocused on the initial element, meaning you can't progress down the page.

**Example:**
Go to the [Timber demo site](https://timber-demo.myshopify.com/) and use the keyboard to tab. Result: Fails after 'Catalog'.

**Comment by cshold** [comment]
@stevebosworth I know you were looking into this on one of our free themes - Solo maybe?
